cpp/src/prims/detail/per_v_transform_reduce_e.cuh

```diff
@@ -589,6 +589,12 @@ __global__ static void per_v_transform_reduce_e_mid_degree(
       }
     }
 
+#if CCCL_MAJOR_VERSION >= 3
```

Comment:
Could you add the following comment:

// FIXME: Remove CUB reference once upgraded to CCCL version 3.x

or something like that.  We use FIXME comments to remind ourselves of cleanup work to do later.